test/fixtures/responses/index.js

```diff
@@ -61,6 +65,7 @@ module.exports = {
   },
   getTrustlines: {
     filtered: require('./get-trustlines.json'),
+    moreThan400Items: moreThan400Items(require('./trustline-item.json')),
```

Comment:
One more refinement - how about this:
`moreThan400Items: buildList(require('./trustline-item.json'), 401)`

This keeps our builder generic and places all concerns in one place.